keyboards/lck75/info.txt

```diff
@@ -5,8 +5,11 @@
     "width": 17, 
     "height": 6, 
     "layouts": {
-        "LAYOUT": {
```

Comment:
This file is named `keyboards/lck75/info.txt` where it should be `keyboards/lck75/info.json`. You can see it failing still to render in the lint job output,
```console
```

Reply:
changed JSON filename